src/pages/createOrder.js

```diff
@@ -190,6 +192,7 @@ class CreateOrderPage extends React.Component {
       stepDetails: get(state.stepDetails, 'totalAmount')
         ? state.stepDetails
         : this.getDefaultStepDetails(this.props.data.Tier),
+      customData: this.props.customData, // May needs parsing with JSON.parse
```

Comment:
The comment is confusing. It needs or it doesn't needs?

Reply:
It needs parsing because we need to be sure the data from the url is structured as expected to avoid breaking anything. Also data sent from querystring are basically strings, parsing it makes it a usable json object for the consumer, I was hoping to leave that to the consumer to parse. But I will do it here because it is more important.

Reply:
@flickz What's the status here? I'm not sure what/when are the next steps?